zebracrossing/campaigns/tests.py

```diff
@@ -41,24 +42,30 @@ def setUp(self):
             cost=23000,
         )
         self.booking_sheet.save()
+
         self.time_slot = TimeSlot(time=datetime.time(hour=12, minute=53))
         self.time_slot.save()
+
         self.booked_day = BookedDay.objects.create(
-            date=datetime.date(2022, 6, 25), timeslot=self.time_slot, bookingsheet=self.booking_sheet
+            date=datetime.date(2022, 6, 25),
+            timeslot=self.time_slot,
+            bookingsheet=self.booking_sheet,
         )
         self.booked_day.save()
-    
+
     def test_save_to_table(self):
-        try:
-            # use post method here and give it data
-            response = self.client.get(self.save_to_table_url)
-        except json.decoder.JSONDecodeError:
-            pass
-                
-        self.assertEquals(str(self.booked_day.timeslot), "12:53")
-        self.assertEquals(str(self.booked_day.date), "2022-06-25")
-        self.assertEquals(self.booked_day.bookingsheet.cost, 23000)
-    
+        dict_expected = {
+            "slot_time": "12:53",
+            "date": "Saturday (25/06)",
+            "bookingsheet_id": f"{self.booking_sheet.id}",
+        }
+        response = self.client.post(self.save_to_table_url, dict_expected, follow=True)
+
+        self.assertEqual(str(self.booked_day.timeslot), "12:53")
```

Comment:
These assertions should be new instances fetched from the DB against `self.booked_day.timeslot`